css/popup.css

```diff
@@ -0,0 +1,162 @@
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+a {
+  color: #FFF;
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0px;
+  padding-left: 0px;
+}
+
+ul li span {
+  float: right;
+  margin-top: 9px;
+  font-size: 13px !important;
+  border: 2px solid rgba(255, 255, 255, 0.7);
+  display: inline-block;
+  text-align: center;
+  height: 30px;
+  font-weight: bold;
+  width: 30px;
+  line-height: 25px;
```

Comment:
haven't seen the line-height used a lot to define the height of an element, not sure if it is ok to use it for this type of tasks, i rarely use it, just when there is a lot of text and i want to make the line-height of it smaller.
Instead i'm using padding to make it bigger,